modules/rtbhouseBidAdapter.js

```diff
@@ -96,14 +96,25 @@ function mapImpression(slot) {
     id: slot.bidId,
     banner: mapBanner(slot),
     native: mapNative(slot),
-    tagid: slot.adUnitCode.toString()
+    tagid: slot.adUnitCode.toString(),
   };
 
   const bidfloor = parseFloat(slot.params.bidfloor);
   if (bidfloor) {
     imp.bidfloor = bidfloor
   }
 
+  const viewability = parseFloat(slot.params.viewability);
+  if (viewability) {
+    imp.metric = [{
+      type: 'viewability',
+      vendor: 'properietary',
+      value: isNaN(viewability)
```

Comment:
To się nie ma szansy zdarzyć wg mnie, NaN nie przejdzie if-a wyżej, ale jako defensive ew. może zostać.

Reply:
Potraktowałem to jako dokumentację bardziej, ale fakt, nie powinno się nigdy zdarzyć po ifie wyżej. Dzięki za uwagę!